iocore/net/QUICPacketHandler.cc

```diff
@@ -62,10 +62,15 @@ QUICPacketHandler::_read_connection_id(IOBufferBlock *block)
 QUICPacketHandlerIn::QUICPacketHandlerIn(const NetProcessor::AcceptOptions &opt, SSL_CTX *ctx) : NetAccept(opt), _ssl_ctx(ctx)
 {
   this->mutex = new_ProxyMutex();
+  // create Connection Table
+  QUICConfig::scoped_config params;
+  _ctable = new QUICConnectionTable(params->connection_table_size());
 }
 
 QUICPacketHandlerIn::~QUICPacketHandlerIn()
 {
+  // TODO: clear all values before destory the table.
+  delete _ctable;
```

Comment:
Rather than an explicit delete, why not use `std::unique_ptr`?

Reply:
It's a kind of historical reason. `std::vec` doesn't work correctly if you allocate a memory with THREAD_ALLOC or something like that for a object that has `std::vec` because it doesn't call constructors. Before I realize this, I used new/delete as a workaround. (and now QUIC code uses "placement new" after allocating memory).

So we can use `std::unique_ptr`, but actually we don't need to create it dynamically anymore. I'll fix it later.